worlds/mlss/Client.py

```diff
@@ -23,8 +23,9 @@ class MLSSClient(BizHawkClient):
     local_checked_locations: Set[int]
     goal_flag: int
     rom_slot_name: Optional[str]
-    eCount: int
     eUsed: List[int]
+    room: int
+    local_events: List[int]
```

Comment:
Small thing that I will let go because it's probably not actually an issue:

This should probably be a set, because it's only used for lookup and sets have instant to logarithmic lookup